src/wasm.h

```diff
@@ -570,6 +570,11 @@ enum RefAsOp {
   RefAsI31,
 };
 
+enum ExternConversionOp {
+  Externalize,
+  Internalize,
+};
```

Comment:
Another option might be to make this part of `RefAs`. This does receive a reference and return a reference, so it has the same shape, and is really quite similar aside from the name?

I don't feel strongly, though, and there is obviously some benefit to following the spec names.

Reply:
Yeah, actually I think I will just make these `RefAs` operations. They're similar enough and I don't know that the extra class is pulling its weight.